src/renderers/native/ReactNativeTypes.js

```diff
@@ -53,6 +53,19 @@ export type NativeMethodsMixinType = {
   setNativeProps(nativeProps: Object): void,
 };
 
+export interface IReactNativeComponent {
+  blur(): void,
+  focus(): void,
+  measure(callback: MeasureOnSuccessCallback): void,
+  measureInWindow(callback: MeasureInWindowOnSuccessCallback): void,
+  measureLayout(
+    relativeToNativeNode: number,
+    onSuccess: MeasureLayoutOnSuccessCallback,
+    onFail: () => void,
+  ): void,
+  setNativeProps(nativeProps: Object): void,
+}
```

Comment:
I'd prefer to use `NativeMethodsMixinType` and add a "TODO" comment to replace it with an interface once `NativeMethodsMixin` is gone.